### PR TITLE
[Translation] fix german translation (spelling, commas)

### PR DESCRIPTION
**Pull request by nikolockenvitz**
* fix spelling mistakes and comma placement / missing commas
* use hyphens in compounds (for the German speaking people, checkout those two Wikipedia articles [[1]](https://de.wikipedia.org/wiki/Komposition_(Grammatik)#Rechtschreibung) [[2]](https://de.wikipedia.org/wiki/Deppenleerzeichen))
* add translation for `hint_show_in_total`

I may have not caught all the errors, but there should be significantly fewer now.

### CODE REVIEW

File: app/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -236,19 +237,19 @@
...
     <string name="about_hint_support">Unterstütze die Entwicklung</string>
-    <string name="about_hint_website">Website</string>
+    <string name="about_hint_website">Webseite</string>
     <string name="activity_donation_action_share_message">Hi, schau dir %s an</string>
     <string name="activity_donation_action_share_using">Teile über</string>
     <string name="activity_donation_crypto_address_copied">Adresse kopiert</string>
-    <string name="activity_intro_title_slide_4">Cloud Sicherungen</string>
-    <string name="activity_launcher_first_start_button_first_time">Erste mal</string>
+    <string name="activity_intro_title_slide_4">Cloud-Sicherungen</string>
+    <string name="activity_launcher_first_start_button_first_time">Erstes mal</string>
```
Comment (nmxcgeo) [inline app/src/main/res/values-de/strings.xml:252]: I'd capitalize "mal".

File: app/src/main/res/values-de/strings.xml [hunk 2]
```diff
@@ -301,14 +302,14 @@
...
-    <string name="empty_message_secondary_panel">Wähle ein Element von der linken Seite für mehr details aus</string>
+    <string name="empty_message_secondary_panel">Wähle ein Element von der linken Seite für mehr Details aus</string>
     <string name="error_input_currency_not_valid">Keine gültige Währung</string>
     <string name="error_input_date_range_not_valid">Kein gültiger Datumsbereich</string>
     <string name="error_input_decimals_not_valid">Muss zwischen 0 und 8 liegen</string>
     <string name="error_input_invalid_date_range">Kein gültiger Datumsbereich</string>
     <string name="error_input_invalid_multiple_wallets">Geldbörsen müssen die gleiche Währung besitzen</string>
     <string name="error_input_iso_not_valid">Kein gültiger ISO-Code</string>
-    <string name="error_input_missing_budget_type">Wähle ein Bugettyp</string>
+    <string name="error_input_missing_budget_type">Wähle einen Bugettyp</string>
```
Comment (nmxcgeo) [inline app/src/main/res/values-de/strings.xml:312]: There is a typo in "Bugettyp" -> "Budgettyp".

**Review by nmxcgeo — APPROVED** [review]
This pull request mostly extends #116.
One thing more I did there was to change line 160 from "Kategorie nicht gefunden" to "Keine Kategorie gefunden". 

If you like that change, I can withdraw my pull request.

Keeping my two comments in mind, your changes look good to me.

**Comment by nikolockenvitz** [comment]
Hi @nmxcgeo, thanks for your feedback and great that you spotted two mistakes here. It absolutely makes sense to merge the PRs into one as it seems to take some time here to get PRs merged. If you are fine with my changes, I would suggest to merge mine into your branch/PR. I can quickly fix the mentioned mistakes and update this PR base to your branch if you like.

**Comment by nmxcgeo** [comment]
Feel free to include my changes into your pull request. I will then close mine.

Greetings Nmxcgeo

**Comment by nikolockenvitz** [comment]
@nmxcgeo I merged your branch/commit into this one here, so that all your changes of #116 are in here now too. Thanks again for your feedback and also for submitting your PR some time ago.

@wandertaker @elgorro thanks for taking the time to review nmxcgeos PR. You may also want to have a look at this PR. Feel free to point out any mistakes I've made or overlooked.

**Comment by nikolockenvitz** [comment]
@AndreAle94 could you please merge this PR and release a new version on PlayStore/F-Droid? My PR fixes a lot of mistakes in the German translation, among them a few critical ones (like saying "ManeyWollet" instead of "MoneyWallet"). Thx

**Review by elgorro — APPROVED** [review]
+Kudos

**Comment by AndreAle94** [comment]
Thank you!

**Pull request closed without merging**